app/bundles/InstallBundle/Views/Install/check.html.twig

```diff
@@ -0,0 +1,104 @@
+{#
+    @copyright   2014 Mautic Contributors. All rights reserved
+    @author      Mautic
+    @link        http://mautic.org
+    @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+#}
+
```

Comment:
I was hoping it will be removed in all places, but we can do it any time :)

Just please do not add these comments going forward.

Reply:
@JoshuaEstes please consider this note.

@escopecz we can do a simple find and replace to remove them